tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "declaration": true,
     "outDir": "lib",
     "noEmit": true,
-    "jsx": "react-native",
+    "jsx": "react-jsx",
```

Comment:
je ne sais pas si tu avais ça aussi, mais j'avais pas mal de pages (LearnerCourseProfile par exemple avec une erreur "React refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686)" sur les balises <> alors que depuis react 17 React est par défaut inclus et on a pas besoin de l'importer. De ce que j'ai cru comprendre "jsx": "react-native" ne prends pas en charge automatiquement la transformation jsx sur web comme le fait react-jsx d'où l'erreur et ce changement

Reply:
Non je n'ai pas remarqué cette erreur. Tu l'as quand tu arrives sur la page d'une formation ?

Reply:
directement dans vs code c'est une erreur eslint sur des fichiers comme TrainercoursePRofile, learnerCourseProfile par exemple

Reply:
Ah oui mon linter ne fonctionnait pas visiblement... En effet, y avait bien l'erreur sur la branche dev

Reply:
Tu as regardé quels sont lse impacts d'un tel changement ? Est-ce qu'on doit pas tester toute l'app ?

Reply:
mmh c'est peut-être pas une idée de ouf sur une app react-native en effet. Ca a l'air de faire des opti en prod mais c'est recommandé d'utiliser react-native sur les apps react-native.
La règle tslint en question on l'a désactivée sinon il faut mettre React dans tous les fichiers ce qui est un peu chiant. Je sais pas pourquoi l'erreur ressort quand même sur certains fichiers. Je propose qu'on attende la prochaine maj expo et typescript pour voir si on veut faire cette modif